chapters/syntax.tex

```diff
@@ -157,7 +157,7 @@ \subsection{Class Definition}\doublelabel{class-definition}
 \subsection{Extends}\doublelabel{extends}
 \begin{lstlisting}[language=grammar]
 extends-clause :
-   extends type-specifier [ class-modification ] [annotation]
+   extends type-specifier [ class-modification ] [ annotation ]
```

Comment:
Should `annotation` be the new (poorly named) `annotation-comment` here?

Reply:
Yes, that was handled in https://github.com/modelica/ModelicaSpecification/pull/2428